### PR TITLE
Download and run the Pantheon WordPress Upstream tests

### PR DESCRIPTION
**Pull request by danielbachhuber**
See https://github.com/pantheon-systems/pantheon-wordpress-upstream/issues/6

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@stevector #reviewmerge

File: bin/behat-prepare.sh [hunk 1]
```diff
@@ -66,3 +66,18 @@ git push
 terminus wp "core install --title=$TERMINUS_ENV-$TERMINUS_SITE --url=$PANTHEON_SITE_URL --admin_user=pantheon --admin_email=wp-redis@getpantheon.com --admin_password=pantheon"
 terminus wp "cache flush"
 terminus wp "plugin activate wp-redis"
+
+###
+# Download the Pantheon WordPress Upstream tests
+###
+cd $BASH_DIR/..
```
Comment (stevector) [inline bin/behat-prepare.sh]: Let's move the downloading of these tests to `require-dev` of composer.json. Behat itself is already there.

Comment (danielbachhuber) [inline bin/behat-prepare.sh]: How does that work?

Comment (stevector) [inline bin/behat-prepare.sh]: https://github.com/pantheon-systems/wp-redis/pull/124

**Pull request merged**